test-plan.md

```diff
@@ -94,7 +94,13 @@ The easist way to verify that a project was successfully loaded is to open a .cs
 
 #### Omnisharp Options
 
-  #### omnisharp.useMono (for Linux)
+  #### omnisharp.useGlobalMono (for Linux/Mac)
+  This option can be set to any of the following values:
+  * "auto" - Will launch OmniSharp using mono if version>=5.2.0 is installed but will launch using the run script if that is not so.
+  * "always" - Will launch OmniSharp using mono if version>=5.2.0 is installed and will throw an error otherwise.
+  * "never" - Launches OmniSharp without using the global mono
+
+  The value of OmniSharp path displayed in the OmniSharp log can be used to know if OmniSharp has launched using mono or not. If it is running using global mono, the path will end with "OmniSharp.exe" else the path will end with "run". 
```

Comment:
Looks good! FWIW, the OmniSharp Log will also list the version of Mono that OmniSharp is running on if it runs on a globally-installed Mono. Not sure if that's interesting to mention here.